app/models/manageiq/providers/redhat/infra_manager/event_fetcher.rb

```diff
@@ -313,6 +313,7 @@ def set_event_name!(event)
     602  => "USER_HOST_MAINTENANCE_MIGRATION_FAILED",
     603  => "HOST_SET_NONOPERATIONAL_IFACE_DOWN",
     605  => "HOST_USED_AS_PROXY",
+    620  => "USER_HOST_MAINTENANCE",
```

Comment:
while at it, I'd include the other host maintenance event types (added for host-engine's host maintenance):

- VDS_MAINTENANCE_MANUAL_HA (10452)
- USER_VDS_MAINTENANCE_MANUAL_HA (10453)